### DISCUSSION TITLE
Help Center Search

### DISCUSSION THREAD

**Discussion by marc-hermann** [cloudbees/zendesk-java-client#533] [Q&A] [answered]
Hi guys,

I am just starting to get familiar with ZenDesk and your API, so please forgive me if I ask something that others have asked before.
I went through your API and noticed that the search for helpdesk articles is missing. I also noticed that the link for the search implementation leads to a 404.
Therefore my question: Is the Help Center Search API implemented in this java API ?

**Comment by PierreBtz** [comment, marked as answer]
@marc-hermann hello, unsure what exactly you are looking for, I found https://github.com/cloudbees-oss/zendesk-java-client/blob/5b633c37a9587c33ed45b6b87a359f7ed56c205d/src/main/java/org/zendesk/client/v2/Zendesk.java#L488-L491
